cli/batch/help.py

```diff
@@ -0,0 +1,4 @@
+"""
+Provide help messages for command line interface's batch commands.
+"""
+GET_BATCH_ID_ARGUMENT_HELP_MESSAGE = 'Get batch by identifier.'
```

Comment:
`GET_BATCH_ID_ARGUMENT_HELP_MESSAGE` > `BATCH_ID_ARGUMENT_HELP_MESSAGE`.

`Get batch by identifier` > `Identifier to get a batch by`.

Reply:
Done.